src/MakersOfDenmark.WebAPI/Controllers/MakerSpaceController.cs

```diff
@@ -53,10 +53,10 @@ public async Task<IActionResult> Get(Guid id)
 
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
-        [HttpGet("{search}")]
-        public async Task<IActionResult> GetSelectionOfMakerspaces(string name)
+        [HttpGet("SearchForMakerSpace/{name}")]
```

Comment:
Jeg vil ikke mene at det skal hedde SearchForMakerSpace.

Så kommer endpointet til at se således ud:

MakerSpace/SearchForMakerSpace. Controlleren sætter alle konteksten for hvad du søger efter.

Jeg vil også mene at name skal være en [FromQuery] og ikke være via routen.

Reply:
Det sørger jeg for at refaktorere (query-string mm.). Men kan det ikke skabe uklarhed, hvis vi får flere søgefunktionaliteter, f.eks. søg efter brugere? Så har vi to endpoints som potentiel hedder "/search", hvor det ville være mere klart med "/SearchForMakerSpace" og "/SearchForUser". Men det er måske at tænke for langt frem (YAGNI)?

Reply:
Nej, så har Usercontroller bare en søge funktion. 

User controllerens søge funktion får følgende URL "user/search"

Reply:
Ja, det giver jo god mening. +1 bliver ændret